src/components/errorbars/plot.js

```diff
@@ -60,6 +61,11 @@ module.exports = function plot(traces, plotinfo, transitionOpts) {
                 .style('opacity', 1);
         }
 
+        errorbars.call(
+            Drawing.setClipUrl,
+            plotinfo._hasClipOnAxisFalse ? plotinfo.clipId : null
+        );
```

Comment:
perhaps it would simplify things if when we calculate `_hasClipOnAxisFalse` we stash `plotClipId` and `layerClipId` (each of which is either `clipId` or `null`)?

Also a debatable style point, I've started avoiding `.call` when it's not part of a method chain - so this would just be `Drawing.setClipUrl(errorbars, clipId)`